_inc/client/state/site-verify/actions.js

```diff
@@ -46,6 +46,8 @@ export const verifySiteGoogle = () => {
 		return restApi.verifySiteGoogle().then( data => {
 			dispatch( {
 				verified: data.verified,
+				google_search_console_url: data.google_search_console_url,
+				google_verification_console_url: data.google_verification_console_url,
```

Comment:
Shouldn't we snakecase these properties? They will end up in the global state tree.